apptest.go

```diff
@@ -177,6 +180,52 @@ func (a *AppSetup) InstallApps(ctx context.Context, apps []App) error {
 	return nil
 }
 
+func (a *AppSetup) UpgradeApp(ctx context.Context, current, desired App) error {
+	var err error
+
+	err = a.createAppCatalogs(ctx, []App{current, desired})
+	if err != nil {
+		return microerror.Mask(err)
+	}
+
+	// if current version have no specific version, use the latest instead.
+	if current.Version == "" && current.SHA == "" {
```

Comment:
if the current version doesn't have any version, it makes sense to use the latest one. 
This is comfortable from other repos. otherwise, the integration test should use `appcatalog` pkg and have to know the appcatalog URL beforehand.

Reply:
Nice, yes we just want the repo being tested to use apptest and not appcatalog as well.